compaction/compaction_manager.cc

```diff
@@ -1955,6 +1997,16 @@ future<> compaction_manager::perform_sstable_upgrade(owned_ranges_ptr sorted_own
     return rewrite_sstables(t, sstables::compaction_type_options::make_upgrade(), std::move(sorted_owned_ranges), std::move(get_sstables), info).discard_result();
 }
 
+future<compaction_manager::compaction_stats_opt> compaction_manager::perform_split_compaction(table_state& t, sstables::compaction_type_options::split opt, std::optional<tasks::task_info> info) {
+    auto get_sstables = [this, &t] {
+        return make_ready_future<std::vector<sstables::shared_sstable>>(get_candidates(t));
+    };
+    owned_ranges_ptr owned_ranges_ptr = {};
+    auto options = sstables::compaction_type_options::make_split(std::move(opt.classifier));
+
+    return perform_task_on_all_files<split_compaction_task_executor>(info, t, std::move(options), std::move(owned_ranges_ptr), std::move(get_sstables));
```

Comment:
Does this create a task per sstable? Can't tell from the name.

Reply:
No, one task per table